.buildkite/scripts/run_models.sh

```diff
@@ -21,4 +21,6 @@ dbt run --target "$db" --full-refresh
 dbt test --target "$db"
 dbt run --vars '{using_schedules: false, using_domain_names: false, using_user_tags: false, using_ticket_form_history: false, using_organization_tags: false}' --target "$db" --full-refresh
 dbt test --target "$db"
+dbt run --vars '{using_schedule_histories: true, using_holidays: false}' --target "$db" --full-refresh
```

Comment:
One followup question tho -- should we have a run with using_schedule_histories: true and using_holidays: true?